<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Menadżer Klientów</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/app/icons/res/Locked.png</normaloff>:/app/icons/res/Locked.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionConnectToDatabase"/>
    <addaction name="actionDisconnectFromDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Edycja</string>
    </property>
    <addaction name="actionAddNewClient"/>
    <addaction name="actionDeleteSelectedRows"/>
    <addaction name="separator"/>
    <addaction name="actionApplyChanges"/>
    <addaction name="actionRollbackChanges"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionInformation"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuEdycja"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnectToDatabase"/>
   <addaction name="actionDisconnectFromDatabase"/>
   <addaction name="separator"/>
   <addaction name="actionAddNewClient"/>
   <addaction name="actionDeleteSelectedRows"/>
   <addaction name="separator"/>
   <addaction name="actionApplyChanges"/>
   <addaction name="actionRollbackChanges"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddNewClient">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/Plus Large.png</normaloff>:/app/icons/res/Plus Large.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj nowego klienta</string>
   </property>
   <property name="toolTip">
    <string>Dodaj nowego klienta</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionConnectToDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/Unlocked.png</normaloff>:/app/icons/res/Unlocked.png</iconset>
   </property>
   <property name="text">
    <string>Połącz z bazą danych</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnectFromDatabase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/Locked.png</normaloff>:/app/icons/res/Locked.png</iconset>
   </property>
   <property name="text">
    <string>Odłącz od bazy danych</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionApplyChanges">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/Checkmark.png</normaloff>:/app/icons/res/Checkmark.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz zmiany</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDeleteSelectedRows">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/Minus Large.png</normaloff>:/app/icons/res/Minus Large.png</iconset>
   </property>
   <property name="text">
    <string>Usuń Zaznaczone Wiersze</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/X.png</normaloff>:/app/icons/res/X.png</iconset>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionRollbackChanges">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/No Entry.png</normaloff>:/app/icons/res/No Entry.png</iconset>
   </property>
   <property name="text">
    <string>Cofnij Zmiany</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/app/icons/res/Ticket.png</normaloff>:/app/icons/res/Ticket.png</iconset>
   </property>
   <property name="text">
    <string>Informacje</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
